### PR TITLE
Add DatasetLinks to dataset_description.json

### PR DESCRIPTION
**Pull request by tsalo**
Related to #69.

## Changes proposed in this pull request

- Build dictionary of dataset links and add them to the `dataset_description.json` file.

### CODE REVIEW

File: qsirecon/utils/bids.py [hunk 1]
```diff
@@ -141,59 +142,92 @@ def collect_participants(bids_dir, participant_label=None, strict=False, bids_va
...
-    if "QSIRECON_DOCKER_TAG" in os.environ:
-        desc["DockerHubContainerTag"] = os.environ["QSIRECON_DOCKER_TAG"]
-    if "QSIRECON_SINGULARITY_URL" in os.environ:
-        singularity_url = os.environ["QSIRECON_SINGULARITY_URL"]
-        desc["SingularityContainerURL"] = singularity_url
-        try:
-            desc["SingularityContainerMD5"] = _get_shub_version(singularity_url)
-        except ValueError:
-            pass
+    if "FMRIPREP_DOCKER_TAG" in os.environ:
```
Comment (mattcieslak) [inline qsirecon/utils/bids.py:185]:
```suggestion
    if "QSIPREP_DOCKER_TAG" in os.environ:
```

File: qsirecon/utils/bids.py [hunk 2]
```diff
@@ -141,59 +142,92 @@ def collect_participants(bids_dir, participant_label=None, strict=False, bids_va
...
+        dset_desc["GeneratedBy"][0]["Container"] = {
+            "Type": "docker",
+            "Tag": f"nipreps/fmriprep:{os.environ['FMRIPREP_DOCKER_TAG']}",
```
Comment (mattcieslak) [inline qsirecon/utils/bids.py:188]:
```suggestion
            "Tag": f"pennlinc/qsirecon:{os.environ['QSIPREP_DOCKER_TAG']}",
```

Comment (mattcieslak) [reply qsirecon/utils/bids.py:188]: I'm actually not sure whether these should be qsiprep or qsirecon - is it for the input data or the data processed here?

Comment (tsalo) [reply qsirecon/utils/bids.py:188]: It's for the environment used by QSIRecon. I'll fix it.

**Review by mattcieslak — APPROVED** [review]
Excellent!!

**Pull request merged**